plugins/mac-virtualcam/src/camera-extension/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+foreach(_uuid IN ITEMS VIRTUALCAM_DEVICE_UUID VIRTUALCAM_STREAM_UUID VIRTUALCAM_SINK_UUID)
```

Comment:
What do you think about renaming `VIRTUALCAM_STREAM_UUID` to `VIRTUALCAM_SOURCE_UUID`?
I made such a change on my branch, probably after you looked at it. Both source and sink are streams so source_uuid would be more precise I think.

Reply:
The UUID is used for a `CMIOExtensionStreamSource`, so it's important to keep the name of the UUIDs matched with their purpose. Calling it a "source" UUID removes that logical connection.

Reply:
In that case, both should contain `_STREAM_`, no? Both are just streams for the same device on different indexes (0 for the source stream and 1 for the sink stream) and directions.

Reply:
Err, you are right and I'm wrong - if you align it with the Swift classes, it should be `SOURCE` then.